rustls/src/crypto/ring/tls13.rs

```diff
@@ -0,0 +1,204 @@
+use crate::crypto::cipher::{
+    make_tls13_aad, AeadKey, Iv, MessageDecrypter, MessageEncrypter, Nonce,
+};
+use crate::enums::{CipherSuite, ContentType, ProtocolVersion};
+use crate::error::Error;
+use crate::msgs::base::Payload;
+use crate::msgs::codec::Codec;
+use crate::msgs::message::{BorrowedPlainMessage, OpaqueMessage, PlainMessage};
+use crate::suites::BulkAlgorithm;
+#[cfg(feature = "secret_extraction")]
+use crate::suites::ConnectionTrafficSecrets;
+use crate::suites::{CipherSuiteCommon, SupportedCipherSuite};
+use crate::tls13::{Tls13AeadAlgorithm, Tls13CipherSuite};
+
+use ring::aead;
+
+/// The TLS1.3 ciphersuite TLS_CHACHA20_POLY1305_SHA256
+pub static TLS13_CHACHA20_POLY1305_SHA256: SupportedCipherSuite =
```

Comment:
Do these keep the `TLS13` prefix that stutters with the module name because of the re-export into `cipher_suites` in `lib.rs`? 

If so, should the `tls12` ciphersuites keep a similar prefix? I've been leaving other feedback that recommends dropping the stutter but I can see how the `cipher_suite` module might change the calculus for that recommendation.

Reply:
these names are intended to echo the standard name (in this case) `TLS_CHACHA20_POLY1305_SHA256`. i think we could make a follow-up to this that makes them (and the other constants around here) actually match the standard IANA names?

Reply:
> i think we could make a follow-up to this that makes them (and the other constants around here) actually match the standard IANA names?

That sounds good to me :+1: